src/YieldDistributor.sol

```diff
@@ -236,6 +259,8 @@ contract YieldDistributor is OwnableUpgradeable {
     /**
      * @notice Internal function for casting votes for a specified user
      * @param _account Address of user to cast votes for
+     *  /* Unless I'm missing something, calling these basis points is pretty confusing, since that term
+     *  usually refers to 1/100th of a percent, which these are not.
```

Comment:
these points can be up to 10000, does that make sense?

Reply:
That's param-controlled and changeable, right? So they happen to be basis points currently but that is contingent. That does explain it though.